reference/gdscript.rst

```diff
@@ -696,6 +696,8 @@ in the loop variable.
     for i in range(2,8,2):
         statement  # similar to [2, 4, 6] but does not allocate an array
 
+    for c in "Hello":
+        print(c)   # iterate through all characters in a String, print every letter on new line
 match
```

Comment:
Would have been nice to keep a newline before the `match` section... Hope sphinx doesn't mess up the html rendering.